templates/individual_response/post-confirmation.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'individual_response/question.html' %}
+
+{% block after_submit_button_content %}
+    {% if is_post_confirm_page %}
+      <p class='u-mt-xl'>{{ _("To request a census in a different format or for further help, please <a href='{contact_us_url}'>contact us</a>".format(contact_us_url=contact_us_url)) }}</p>
+    {% endif %}
```

Comment:
In the prototype, this text and link are only on the "how" page. Also, the text doesn't appear on screen as it's still wrapped in a conditional that is never set.